tox.ini

```diff
@@ -45,17 +45,18 @@ passenv =
 deps =
     {tests,clean,coverage}: coverage
     {tests,functests,lint}: httpretty
-    {tests,functests,lint,docstrings,checkdocstrings}: pytest
+    {tests,bddtests,functests,lint,docstrings,checkdocstrings}: pytest
```

Comment:
This is needed because we load some settings from a `pytest` conf file, which happens to load `pytest` even though we don't use it.